src/intTest/resources/edifact/pathology.fhir.json

```diff
@@ -64,6 +64,23 @@
         ]
       }
     },
+    {
+      "fullUrl": "urn:uuid:8be835ba-540a-4a02-bf0c-0be947b98c82",
+      "resource": {
+        "resourceType": "ProcedureRequest",
+        "id": "8be835ba-540a-4a02-bf0c-0be947b98c82",
+        "status": "unknown",
+        "intent": "?",
+        "code": {
+          "text": "unknown"
+        },
+        "note": [
+          {
+            "text": "TIRED ALL THE TIME, LOW Hb"
```

Comment:
Can we update the integration test data to test multiple free texts from EDIFACT map to FHIR ok?

Reply:
Sure!
269f9fa7723a216a0cb88adc9301a2668877de6b